### ISSUE TITLE
Decimals

### ISSUE THREAD

**Issue by oneseventhree** [custom-components/sensor.mitemp_bt#58]
In const.py can this be the default on future releases:

DEFAULT_DECIMALS = 1

I don't know why anyone would want 2 decimals but I keep having to change it back each update.

Thank you in advance :)

**Comment by Ernst79** [comment]
You can always set rounding to true and decimals to 1 in your config. That should not change on updating the component.

```
sensor:
  - platform: mitemp_bt
    rounding: True
    decimals: 1
```

**Comment by Magalex2x14** [comment]
I think that since slow sensors like LYWSD03 appeared among the supported ones, the data from which are not averaged over a short `period`, then it’s probably worth, yes, to set 1 decimal as default...

Perhaps it’s worthwhile to indicate in the documentation that the `rounding` option serves for  disable rounding, since it is enabled by default.

**Comment by Magalex2x14** [comment]
Closed with 0.6.4 release.
1 decimal place is the default from now.
